Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -126,6 +126,11 @@ class BrowserViewController: UIViewController {
     override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
```

Comment:
Ya this totally works. I like this better since it keeps the end scroll state unchanged. I updated this PR with a commit that adds this.